docker-compose.yml

```diff
@@ -12,19 +12,22 @@ services:
       MONGO_HOSTNAME: mongo
       MONGO_DB_NAME: fhir
       MONGO_PORT: 27017
-      RESOURCE_SERVER: http://localhost:3000/
-      AUTH_SERVER_URI: http://myauthzserver.com
+      RESOURCE_SERVER: https://staging-icanbwell.auth.us-east-1.amazoncognito.com
```

Comment:
Should this be our server, not the auth server?

Reply:
no, this is the OAuth resource server.  This is not the callback url.

Reply:
In the past, Resource server has been the FHIR resource server.  Auth Server is the separate auth server.  But if they are supposed to be the same I trust you.

Reply:
This isn't really used with Cognito so I reset it to previous values